### PR TITLE
[MNOE-205] Add style to unlock page

### PR DESCRIPTION
**Pull request by hedudelgado**
@x4d3 @ouranos

### CODE REVIEW

File: api/app/views/mno_enterprise/auth/unlocks/_form.html.haml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+= form_for(resource, as: resource_name, url: unlock_path(resource_name), html: { method: :post }) do |f|
+  = devise_error_messages!
+  %div
+    = f.email_field :email, :placeholder => "Email", :required => true, 'ng-model' => 'email', :class => 'form-control'
+  %br
+  %div
+    = f.password_field :password, :placeholder => "Password", :required => true, 'ng-model' => 'password', :class => 'form-control', :autocomplete => "off"
```
Comment (ouranos) [inline api/app/views/mno_enterprise/auth/unlocks/_form.html.haml]: ?

File: api/app/views/mno_enterprise/auth/unlocks/_form.html.haml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+= form_for(resource, as: resource_name, url: unlock_path(resource_name), html: { method: :post }) do |f|
+  = devise_error_messages!
+  %div
+    = f.email_field :email, :placeholder => "Email", :required => true, 'ng-model' => 'email', :class => 'form-control'
```
Comment (ouranos) [inline api/app/views/mno_enterprise/auth/unlocks/_form.html.haml]: Use Ruby 1.9 syntax for hash

https://github.com/bbatsov/ruby-style-guide#hash-literals
https://github.com/bbatsov/ruby-style-guide#no-mixed-hash-syntaces

File: api/app/views/mno_enterprise/auth/unlocks/_form.html.haml [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+  %br
+  %div
+    = f.password_field :password, :placeholder => "Password", :required => true, 'ng-model' => 'password', :class => 'form-control', :autocomplete => "off"
+
+  %br/
+  %div
+    %button{ :class => "btn btn-warning btn-login", :type => "submit", 'ng-click' => 'hasClicked = true', 'ng-disabled' => 'loginForm.$invalid', 'ng-hide' => 'hasClicked' }
```
Comment (ouranos) [inline api/app/views/mno_enterprise/auth/unlocks/_form.html.haml]: I don't think this loginForm is defined anywhere.

**Comment by hedudelgado** [comment]
@ouranos  corrected.
Thanks.

**Comment by ouranos** [comment]
All good, I'll merge it against `3.0` as that can be regarded as a bug introduced back then

**Pull request closed without merging**